### ISSUE TITLE
Fix S1172 FP: Parameter flagged as unused when only used in value tuple

### ISSUE THREAD

**Issue by Montimort** [SonarSource/sonar-dotnet#3803]
When only using a parameter in a value tuple inside the method, it is tagged as 'unused' and removable.
```
// uint id1, uint id2 are tagged as unused here, but they are!
 private void doStuff(uint id1, uint id2, uint value)
        {
            var stack = new Stack<uint>();
            stack.Push(systemElementId);
            _currentIrqStack[(id1, id2)] = stack;
        }
```
VS Version: 15.9.29
Sonarlint version: 4.28.0.24595

**Comment by costin-zaharia-sonarsource** [comment]
I can confirm the FP

Could be related to #3132

**Comment by pavel-mikula-sonarsource** [comment]
This will be fixed by #3255, PR is on it's way
